website/jupyter_nbconvert_config.py

```diff
@@ -0,0 +1,661 @@
+#------------------------------------------------------------------------------
+
+## Base class for Jupyter applications
+
+## Answer yes to any prompts.
+#c.JupyterApp.answer_yes = False
+
+## Full path of a config file.
+#c.JupyterApp.config_file = ''
+
+## Specify a config file to load.
+#c.JupyterApp.config_file_name = ''
+
+## Generate default config file.
+#c.JupyterApp.generate_config = False
+
+#------------------------------------------------------------------------------
+# NbConvertApp(JupyterApp) configuration
+#------------------------------------------------------------------------------
+
+## This application is used to convert notebook files (*.ipynb) to various other
+#  formats.
+#
+#  WARNING: THE COMMANDLINE INTERFACE MAY CHANGE IN FUTURE RELEASES.
+
+## The export format to be used, either one of the built-in formats ['asciidoc',
+#  'custom', 'html', 'latex', 'markdown', 'notebook', 'pdf', 'python', 'rst',
+#  'script', 'slides'] or a dotted object name that represents the import path
+#  for an `Exporter` class
+#c.NbConvertApp.export_format = 'html'
+
+## read a single notebook from stdin.
+#c.NbConvertApp.from_stdin = False
+
+## URL base for ipywidgets package
+#c.NbConvertApp.ipywidgets_base_url = 'https://unpkg.com/'
+
+## List of notebooks to convert. Wildcards are supported. Filenames passed
+#  positionally will be added to the list.
+#c.NbConvertApp.notebooks = []
+
+## overwrite base name use for output files. can only be used when converting one
+#  notebook at a time.
+#c.NbConvertApp.output_base = ''
+
+## Directory to copy extra files (figures) to. '{notebook_name}' in the string
+#  will be converted to notebook basename.
+#c.NbConvertApp.output_files_dir = '{notebook_name}_files'
+
+## PostProcessor class used to write the results of the conversion
+#c.NbConvertApp.postprocessor_class = ''
+
+## Whether to apply a suffix prior to the extension (only relevant when
+#  converting to notebook format). The suffix is determined by the exporter, and
+#  is usually '.nbconvert'.
+#c.NbConvertApp.use_output_suffix = True
+
+## Writer class used to write the  results of the conversion
+#c.NbConvertApp.writer_class = 'FilesWriter'
+
+#------------------------------------------------------------------------------
+# NbConvertBase(LoggingConfigurable) configuration
+#------------------------------------------------------------------------------
+
+## Global configurable class for shared config
+#
+#  Useful for display data priority that might be used by many transformers
+
+## Deprecated default highlight language as of 5.0, please use language_info
+#  metadata instead
+#c.NbConvertBase.default_language = 'ipython'
+
+## An ordered list of preferred output type, the first encountered will usually
+#  be used when converting discarding the others.
+c.NbConvertBase.display_data_priority = ['image/png', 'image/jpeg', 'text/markdown', 'text/plain', 'text/html', 'application/pdf', 'text/latex', 'image/svg+xml']
```

Comment:
For the record: usually text/markdown and text/plain are listed before the image types

Reply:
todo: add a comment with the default value above.

Reply:
done